syntax/htmlok.php

```diff
@@ -22,7 +22,7 @@ public function getPType()
     /** @inheritDoc */
     public function getSort()
     {
-        return 190;
+        return 189;
```

Comment:
While I agree with this change (makes the plugin override the default in pre "Jack Jackrum" DokuWiki), shouldn't this be a separate PR?